### PR TITLE
Feature/ui top panel hydrator++ 5

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Adds top panel to published pipeline
- Fixes the behavior of read only vs editable top panel widgets

Latest Bamboo build: http://builds.cask.co/browse/CDAP-DRC3988-2
##### Note:

There are three modes in which we can use post run action wizard - `create, edit & view`
- Create : Has all three stages and the user can select, configure & confirm. They can navigate between different stages without restriction.
- Edit: Has the configure & confirm stage. Can go back and forth between last 2 stages.
- View: Has the only option of viewing the config in the last stage.

### CODE REVIEW

**Comment by elfenheart** [comment]
Minor improvement suggestion:

for schedule detail view (disabled) mode, should we just hide the basic/advanced radio button?

screen shot 2016-07-13 at 11 57 45 am

**Comment by elfenheart** [comment]
Otherwise 👍 LGTM!!

File: cdap-ui/app/directives/my-post-run-action-wizard/my-post-run-action-wizard.js [hunk 1]
```diff
@@ -31,24 +30,22 @@ angular.module(PKG.name + '.commons')
...
           rStore: () => store,
-          rIsDisabled: () => isDisabled || false,
-          rAction: () => action || null
+          rAction: () => action || null,
+          rMode: () => mode
         },
         size: 'lg',
         windowClass: 'post-run-actions-modal cdap-modal',
-        controller: ['$scope', 'rActionCreator', 'rIsEdit', 'rStore', 'rAction', 'rIsDisabled', function($scope, rActionCreator, rIsEdit, rStore, rAction, rIsDisabled) {
+        controller: ['$scope', 'rActionCreator', 'rStore', 'rMode', 'rAction', function($scope, rActionCreator, rStore, rMode, rAction) {
```
Comment (elfenheart) [inline cdap-ui/app/directives/my-post-run-action-wizard/my-post-run-action-wizard.js:45]: should do `'ngInject';` instead

**Comment by elfenheart** [comment]
👍  LGTM

File: cdap-ui/app/directives/my-post-run-action-wizard/my-post-run-action-wizard.js (hunk 1 above)
Comment (ajainarayanan) [inline cdap-ui/app/directives/my-post-run-action-wizard/my-post-run-action-wizard.js:45]: Talked to @elfenheart. ngAnnotate cannot add angular annotations to anonymous functions. Moving this change to another PR as we have more than few places where we need to fix this to be consistent.

**Comment by ajainarayanan** [comment]
Demoed with @bdmogal.Merging
